homeassistant/components/traccar/device_tracker.py

```diff
@@ -172,11 +159,24 @@ def __init__(self, api, hass, async_see, scan_interval,
             for custom_attr in self._custom_attributes:
                 if device_info.get(custom_attr) is not None:
                     attr[custom_attr] = device_info[custom_attr]
+                    if custom_attr in self._skip_accuracy_on:
+                        skip_accuracy_filter = True
+
+            accuracy = float(0)
+            if device_info.get('accuracy') is not None:
+                accuracy = device_info['accuracy']
+            if (not skip_accuracy_filter and self._max_accuracy > 0 and
```

Comment:
Looks like `self._max_accuracy` can also be `None`, in which case the comparison will fail with an exception. Is that correct?

Reply:
Nvm, the validator has a default.

Reply:
Yes, `self._max_accuracy` is set to `0` by default if it's not defined in configuration. So it shouldn't fail.